.travis.yml

```diff
@@ -154,3 +165,32 @@ script:
   - cmake --build .
   - ctest --output-on-failure
   - make install
+
+
+#=============================================================================
+# Prepare Documentation
+#=============================================================================
+after_success:
+  - if [ "$BUILD_DOCUMENTATION" = "ON" ] && [ "$TRAVIS_OS_NAME" == "linux" ]; then
+      echo "------- Build Doxygen Documentation -------";
+      pushd $TRAVIS_BUILD_DIR/build;
+      make docs;
+      echo "-------- Disable jekyll --------";
+      pushd $TRAVIS_BUILD_DIR/build/docs;
+      touch .nojekyll;
+    fi
+
+
+#=============================================================================
+# Documentation deployment
+#=============================================================================
+deploy:
```

Comment:
@pramodk as far as I can tell, there is no need for extra setup to push documentation to `gh-pages`.

Reply:
yeah, thats what I remember.